assets/js/components/adminbar/AdminBarZeroData.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Admin Bar Zero Data component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const AdminBarZeroData = () => {
+	return (
+		<div className="mdc-layout-grid__cell--span-12">
+			<h3>{ __( 'No data available yet', 'google-site-kit' ) }</h3>
```

Comment:
Let's make this an `h4` instead (h3 looks a bit big).

Also, let's put the text content on its own line like we do in most places.
```suggestion
			<h4>
				{ __( 'No data available yet', 'google-site-kit' ) }
			</h4>
```